### PR TITLE
Dashboard hub

### PR DESCRIPTION
**Pull request by BastinJafari**

### CODE REVIEW

**Comment by BastinJafari** [comment]
I am waiting for the final implementation of the hub to make a gif.

**Comment by FionnD** [comment]
Hey @BastinJafari 

Just trying to get a sense of the situation on some PR. This PR is fully reviewed and ready to be merged once the new hub layout is done so we can an accurate gif?

**Comment by BastinJafari** [comment]
I am currently waiting for Hub to be finished before making gifs, that's correct.  
You guys could already comment on the text if there is some change needed

File: chapters/dashboard/hub.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Jina Hub
+
+**[Jina Hub](https://dashboard.jina.ai/#/hub)** is a centralized registry, where you can share and discover custom and community driven Jina Pods and Apps tailored to specific use cases of neural search.
+
+# Features
+
+-  ##  Upload and Share Images
```
Comment (aga11313) [inline chapters/dashboard/hub.md]: use either headings or bullet points, it looks kind of wierd like this

File: chapters/dashboard/hub.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    Share your custom Jina Pods by simply uploading a Python file, a Dockerfile and a YAML, and leave the rest of the work to us. Building and version control is automatically handled on the cloud side.
+
+-  ##  Cloud Native
+    As Jina Hub images are containerized, they can be easily leveraged in any Jina Flow, even in distributed environments.
+
+-  ##  Open Source
+    Hub images are made and maintained by the community.
```
Comment (aga11313) [inline chapters/dashboard/hub.md]: Add one more sentence, maybe 'You too can contribute to the database of hub images and help your fellow developers achieve their search goals!' to make it look less like we run out of ideas for what to write @lusloher any ideas for completing this?

File: chapters/dashboard/hub.md (hunk 1 above)
Comment (BastinJafari) [reply chapters/dashboard/hub.md]: changed it to headings

File: chapters/dashboard/hub.md (hunk 2 above)
Comment (BastinJafari) [reply chapters/dashboard/hub.md]: Added this and a link to Hub GitHub

**Review by aga11313 — APPROVED** [review]
LGTM, happy to merge after the gif has been added

**Pull request merged**